configure.ac

```diff
@@ -5476,6 +5476,20 @@ then
   [Define if you have struct stat.st_mtimensec])
 fi
 
+# checking for glibc extensions to langinfo.h
+#  _NL_TIME_FIRST_WEEKDAY isn't #define'd, so test it directly
+AC_MSG_CHECKING(for _NL_TIME_FIRST_WEEKDAY)
+AC_CACHE_VAL(ac_cv_have__nl_time_first_weekday,
+AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <langinfo.h>]], [[
+int main(int argc, char *argv[]) { return !!nl_langinfo(_NL_TIME_FIRST_WEEKDAY); }]])],
```

Comment:
The second argument to ``AC_LANG_PROGRAM()`` is the body of a main function, not a complete main function.